iron-collapse.html

```diff
@@ -216,6 +216,9 @@
       // Set the final size.
       if (size === 'auto') {
         this.style[this._dimensionMax] = '';
+      // call _transitionEnd if no transition is performed, styles need to be set
+      } else if (size === startSize) {
+        this._transitionEnd();
       } else {
         this.style[this._dimensionMax] = size;
       }
```

Comment:
That's because there is a bug in `_transitionEnd` which resets the `maxHeight` at the end of the transition for opened `iron-collapse` (see https://github.com/PolymerElements/iron-collapse/issues/65). I guess this requires to store the desired size in some private variables, and ensure that is set in `_transitionEnd`. Something like:

```javascript
updateSize: function(size, animated) {
  // Consider 'auto' as '', to take full size.
  size = size === 'auto' ? '' : size;
  // No change!
  if (this._desiredSize === size) {
    return;
  }
  this._desiredSize = size;
  this._updateTransition(false);
  // all the `willAnimate` things ...
},

// ...

_transitionEnd: function() {
  this.style[this._dimensionMax] = this._desiredSize;
  // [...]
```

I believe it's the responsibility of `updateSize` to call `_transitionEnd` if there will be no animations. `_openedChanged` shouldn't do that. WDYT?